package.js

```diff
@@ -3,13 +3,20 @@ Package.describe({
   name: "practicalmeteor:mocha",
   summary: "Write package tests with mocha and run them in the browser or from the command line with spacejam.",
   git: "https://github.com/practicalmeteor/meteor-mocha.git",
-  version: '2.1.0_7'
+  version: '2.1.1-rc.1',
+  testOnly: true
+});
+
+
+Npm.depends({
```

Comment:
@tmeasday is this required? We intentionally added the browserified mocha.js to both client and server. It creates problems with meteor 1.3 rc.13. See issue I mentioned you in.